backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Integration/Clients/ResourceClient.cs

```diff
@@ -78,41 +83,49 @@ public async Task<List<ServiceResource>> GetResources()
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex, "AccessManagement.UI // ResourceRegistryClient // SearchResources // Exception");
+                _logger.LogError(ex, "AccessManagement.UI // ResourceClient // SearchResources // Exception");
                 throw;
             }
 
             return resources;
         }
 
-        /// <summary>
-        /// Get resources of a given type
-        /// </summary>
-        /// <param name="resourceType"> the resource type</param>
-        /// <returns></returns>
-        public async Task<List<ServiceResource>> GetResources(ResourceType resourceType)
+        /// <inheritdoc />
+        public async Task<OrgList> GetAllResourceOwners()
         {
-            List<ServiceResource> resources = new List<ServiceResource>();
-            ResourceSearch resourceSearch = new ResourceSearch();
-            resourceSearch.ResourceType = resourceType;
-            string endpointUrl = $"search?ResourceType={(int)resourceType}";
+            OrgList resourceOwners = new OrgList();
 
-            HttpResponseMessage response = await _httpClient.GetAsync(endpointUrl);
-            if (response.StatusCode == HttpStatusCode.OK)
+            string endpointUrl = "https://platform.at23.altinn.cloud/resourceregistry/api/v1/resource/orgs";
```

Comment:
Prod env is not up yet I think, so we need to go through AT23 until it is up

Reply:
Yeah, but it shouldn't be hardcoded into using at23. It should use the one corresponding to its current environment. And if that one isn't up yet, then it will just not receive an answer yet (which I think is ok)
You can ofc test with data from at23 locally (like this or using the appsettings.Development file), but I don't think it should be pushed up to main.

Reply:
I fix